event-http/pom.xml

```diff
@@ -71,7 +71,7 @@
 
         <dependency>
```

Comment:
I think we want to remove this and use `jakarta.servlet:jakarta.servlet-api` instead

Reply:
I'm asking the Jetty folks via support about this

Reply:
Confirmed that this is the right dependency